dev-app/src/screens/HomeScreen.tsx

```diff
@@ -80,12 +80,6 @@ export default function HomeScreen() {
             navigation.navigate('ReaderDisplayScreen');
           }}
         />
-        <ListItem
-          title="Store card via readReusableCard"
-          onPress={() => {
-            navigation.navigate('ReadReusableCardScreen');
-          }}
-        />
```

Comment:
can we remove this from example-app as well?

Reply:
OIC，I' ll handle it～. There were no changes to the Example code this time because I thought this change should also quickly form the next SDK version of the strip-terminal-treat-native to be released, and the Example will be updated at that time.

Reply:
oh wait you are so right, example-app is still on the old version 🤦‍♀️ lets not change that in this PR